firestore/tests/unit/test__helpers.py

```diff
@@ -133,64 +133,90 @@ def test_unicode(self):
         self.assertEqual(field_path.parts, ('一', '二', '三'))
 
     def test_to_api_repr_a(self):
```

Comment:
It will be much easier and shorter to write a single test that loops over a list of (input, expected) pairs.